scripts/manage-operator.sh

```diff
@@ -30,7 +30,7 @@ set_tags() {
         else
             AUTHOR=$(jq -r '.refs[0].pulls[0].author' <<< ${CLONEREFS_OPTIONS} | tr -d '[:space:]')
             PULL_PULL_SHA=${PULL_PULL_SHA:-$(jq -r '.refs[0].pulls[0].sha' <<< ${CLONEREFS_OPTIONS} | tr -d '[:space:]')}
-            TAGS=from.${AUTHOR}.${REPO_NAME}.${COMMIT_ID_SUFFIX}
+            TAGS=from.${AUTHOR}.${REPO_NAME}.PR${PULL_NUMBER}.${COMMIT_ID_SUFFIX}
```

Comment:
actually, how is it going to help?

Reply:
1. for better orientation in GH Action builds vs what is used in e2e tests
2. and mainly for more context in quay repos - for example take a look at this: https://quay.io/repository/codeready-toolchain-test/host-operator-index?tab=tags - there is already `from.MatousJobanek.toolchain-e2e.PR346.3832901` which says that the index image was build for a PR created by `MatousJobanek` in `toolchain-e2e` repo with a number `346` and for commit `3832901`

Reply:
ok, thanks for the clarification, although 1. is still a bit obscure to me